src/backend/distributed/sql/citus--11.0-4--11.1-1.sql

```diff
@@ -79,3 +79,59 @@ DROP FUNCTION pg_catalog.get_all_active_transactions(OUT datid oid, OUT process_
 
 DROP FUNCTION pg_catalog.isolate_tenant_to_new_shard(table_name regclass, tenant_id "any", cascade_option text);
 #include "udfs/isolate_tenant_to_new_shard/11.1-1.sql"
+
+CREATE TYPE citus.citus_job_status AS ENUM ('scheduled', 'running', 'finished', 'cancelled', 'failing', 'failed');
+ALTER TYPE citus.citus_job_status SET SCHEMA pg_catalog;
+
+CREATE TABLE citus.pg_dist_background_jobs (
+    job_id bigserial NOT NULL,
+    state pg_catalog.citus_job_status DEFAULT 'scheduled' NOT NULL,
+    job_type name,
+    description text,
```

Comment:
I wonder if a `job_type` would always need to be set?

My thought was that one could omit it, especially once it starts referencing some other catalog for callback functions. However, it might be good indeed to force any system using this to register a `job_type` with Citus.

Same for description? They are both not super required for the system to work, although it might be good for the human if we force them to be set.

Reply:
I would say let's require them for now, and if we find a good use case to have NULL later, we can do that then. Maybe also not give them an empty string default though like I suggested in my first comment. That will force us to put something useful there for now at least.